### PR TITLE
new package: iproute2 5.5.0

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [ ] Latest upstream version.
- [ ] I agree to maintain this package (**required**).
- [ ] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/iproute2/build [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+sed -i 's/-Werror//' Makefile
+
+./configure
```
Comment (dylanaraps) [inline community/iproute2/build]: Missing `--prefix`? Can't `--sbindir` also be set here?

File: community/iproute2/build [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+sed -i 's/-Werror//' Makefile
```
Comment (dylanaraps) [inline community/iproute2/build]: Does this line have to exist?

Comment (a-schaefers) [reply community/iproute2/build]: Nooooooope

File: community/iproute2/build (hunk 1 above)
Comment (a-schaefers) [reply community/iproute2/build]: I just copied arch on that, figured it's part of their base so it'd be a quality pkgbuild ...

Comment (dylanaraps) [reply community/iproute2/build]: Gross: https://github.com/shemminger/iproute2/blob/master/configure

Comment (a-schaefers) [reply community/iproute2/build]: ok well it works either way

File: community/iproute2/build [hunk 3]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin
```
Comment (dylanaraps) [inline community/iproute2/build]: These don't actually work. Default prefix is `/usr` and you've probably installed these to `/usr/sbin`!

File: community/iproute2/build [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+patch -p1 < 0001-make-iproute2-fhs-compliant.patch
+
+./configure \
+    --prefix=/usr
```
Comment (dylanaraps) [inline community/iproute2/build]: You can remove this as well. Just `./configure` is fine.

**Comment by a-schaefers** [comment]
SBINDIR is needed for second make. --sbindir=/usr/bin does nothing on configure, and I am unsure if --prefix=/usr does anything useful or not, however I have added it per your last comment, just incase.

**Comment by a-schaefers** [comment]
Ok got it

**Comment by dylanaraps** [comment]
As I said above, they don't do anything ;)

**Comment by a-schaefers** [comment]
merge me baby

**Pull request merged**